### PR TITLE
Added URL for the image path

### PR DESCRIPTION
**Pull request by tharun143**
Signed-off-by: tharun143 <tharunravuri2000@gmail.com>

### CODE REVIEW

**Comment by tharun143** [comment]
@GeorgLink I know you would be furious on me for creating multiple PRs for different commits. But when I ever I try to install hatstall something happens to my pc. First I lost my local repo and then my python3 and then my ubuntu.  

All these unfortunate situations are happening and I find;t expected these. I hope I added the image path correctly. @klumb @GeorgLink

File: About/former.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+
+# Former Board Members:
+
+
+
+## Jono Bacon
+
+![Jono Bacon](https://github.com/chaoss/website/blob/master/About/images/jono.jpg)
```
Comment (GeorgLink) [inline About/former.md:9]: We need to use the github.io URL that is generated for these pictures. See example: https://raw.githubusercontent.com/chaoss/website/master/CHAOSScon/2020EU/speakers.md

**Comment by GeorgLink** [comment]
> @GeorgLink I know you would be furious on me for creating multiple PRs for different commits. 

No, not at all. It is simply more difficult to review when the PRs constantly change and we loose the history of our conversation.

> I lost my local repo

You can recover from deleting your local repo. You have a copy on GitHub (otherwise, you could not have created a pull request).  You should be able to `git clone` the repo again, make changes and push those changes back to GitHub. By pushing commits to the same branch, you will update the pull request. 

I know git workflow and GitHub workflow have a learning curve but I trust you can figure it out. Feel free to ask for help.

**Comment by tharun143** [comment]
@GeorgLink I have added the URL's can you check the md file.

**Comment by GeorgLink** [comment]
The paths will not work. The part that comes after the domain will need to be different. Can you take another look at the example and see where the file is in the repository and how the github.io location will be different? (e.g., the example has no `blob` in the URL)

**Comment by tharun143** [comment]
@GeorgLink I committed. I think now the URL is correct. Also, today I would like to join the meetings of CHAOSS and want to know about the work culture. Thanks

**Comment by tharun143** [comment]
@GeorgLink thanks.

**Comment by klumb** [comment]
Thanks @tharun143

**Pull request merged**